scene/resources/primitive_meshes.h

```diff
@@ -211,6 +211,7 @@ class PlaneMesh : public PrimitiveMesh {
 	Size2 size = Size2(2.0, 2.0);
 	int subdivide_w = 0;
 	int subdivide_d = 0;
+	Vector3 center_offset = Vector3(0.0, 0.0, 0.0);
```

Comment:
`0, 0, 0` is the default value for `Vector3`, so you don't need to assign anything.